src/graphics/dom-layer.js

```diff
@@ -64,6 +64,18 @@ Crafty._registerLayerTemplate("DOM", {
         Crafty._removeDrawLayerInstance(this);
     },
 
+    _lastParentNode: null,
+    events: {
+        "Freeze":function(){
+            this._lastParentNode = this._div.parentNode;
+            this._div.parentNode.removeChild(this._div);
+        },
+        "Unfreeze":function(){
+            this._lastParentNode.appendChild(this._div);
+            delete this._lastParentNode;
+        }
+    },
```

Comment:
So there is a performance benefit to additionally remove the DOM element from the parent?
Note that the `DOM` component already sets `style.visibility = "hidden";`.